File: Cyril/Dart/Basics/ex25.dart
import 'dart:io';

void main() {
  stdout.write("Height of wall: ");
  int test_h = int.parse(stdin.readLineSync()!);

  stdout.write('Width of wall: ');
  int test_w = int.parse(stdin.readLineSync()!);

  int coverage = 5;

  var result = paint_calc(test_h, test_w, coverage);

  print("You'll need ${result} cans of paint");
}

dynamic paint_calc(height, width, coverage) {
  double numberOfCans = ((height * width) / coverage);
  numberOfCans.round();
  return numberOfCans.toInt();
}

File: Cyril/Python/Basics/ex25.py
def paint_calc(height, width, coverage):
    numberOfCans = (height * width) / coverage
    numberOfCans = round(numberOfCans)
    return int(numberOfCans)

test_h = int(input("Height of wall: "))
test_w = int(input("Width of wall: "))
coverage = 5

result = paint_calc(test_h, test_w, coverage)

print(f"You'll need {result} cans of paint")